common/src/test/java/org/apache/rocketmq/common/AclConfigTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.rocketmq.common;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class AclConfigTest {
+
+    /**
+     * Test to verify that getGlobalWhiteAddrs returns null when no addresses have been set.
```

Comment:
Test classes do not require method comments.

Reply:
Is it to remove all method level comments?

Reply:
Yes, it's not necessary in test class.

Reply:
By the way, why does my check item show "7 workflows awaiting approval"

Reply:
Wait, it's first contribution, need the approval of the committer.

Reply:
Thank you very much, I've modified it

Reply:
Sorry to bother you again, is codecov-commenter triggered automatically or do you need to trigger it manually?
![image](https://github.com/user-attachments/assets/0d09cae5-f2f1-4f3b-b5cb-36357b6f72a0)

Reply:
ci automatically triggered.

Reply:
> ci automatically triggered.

Thank you very much